### ISSUE TITLE
Why does the learning rate get larger and larger in the files of the training logs?

### ISSUE THREAD

**Issue by JiangdongWang123** [SlongLiu/query2labels#42]
It's a fun task! But I have a question. Why does the learning rate get larger and larger in the files of the training logs? I trained my dataset using the training code, and before I had finished 80 epochs, it was over. The learning rate showed an upward trend.
